docker/build_docker.sh

```diff
@@ -418,3 +422,5 @@ for docker_dir in `find $SCRIPT_DIR -maxdepth 1 -mindepth 1 -type  d -print | so
         echo ">>>>>>>>>>>>>>> `date`: Done docker build <<<<<<<<<<<<<"
     fi
 done
+echo "Successfully pushed $PUSHED_DOCKERS"
+echo $PUSHED_DOCKERS > $CIRCLE_ARTIFACTS/pushed_dockers.txt
```

Comment:
Do the `echo` after file creation, or add another echo so we know the file was created successfully.